index.bs

```diff
@@ -783,7 +783,22 @@ spec:css-syntax-3;
     6.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
         ancestors=], and `false` otherwise.
 
-    7.  Run the following steps [=in parallel=]:
+    7.  Let |key| be the result of [=map/getting the key=] of |options|.
+
+    8.  If |key| is {{CredentialRequestOptions/publicKey}} then
+        if |settings|' [=responsible document=] is **not** [=allowed to use=] the
+        <a data-lt="publickey-credentials-get-feature">publickey-credentials-get</a>
+        [=policy-controlled feature=] [[PERMISSIONS-POLICY]] return [=a promise rejected with=] `NotSupportedError`.
```

Comment:
You don't really need the `[[PERMISSIONS-POLICY]]` citation since `[=policy-controlled feature=]` links directly there and makes sure it's included in the references.
```suggestion
        [=policy-controlled feature=] return [=a promise rejected with=] `NotSupportedError`.
```

Reply:
https://github.com/w3c/webappsec-permissions-policy/issues/396 suggests `NotAllowedError` or consistency with other exceptions thrown by the interface. The PasswordCredential already throws a `NotAllowedError`, while PublicKeyCredentials throw both that and `NotSupportedError`. Altogether, I think that implies

```suggestion
        [=policy-controlled feature=] [[PERMISSIONS-POLICY]] return [=a promise rejected with=] a "{{NotAllowedError}}" {{DOMException}}.
```

Reply:
Ok, thx, good suggestions, I've incorporated them by hand since the github UI was acting weird (I think because both suggestions are on the same line) and I was not sure what it was going to do.

Reply:
I'm not familiar enough with the implementation to confidently confirm that, but I do read the code the same way you did.

Reply:
Yes that is correct.

Reply:
Ok, thanks folks, this question is now confirmed, am resolving this conversation.